### PR TITLE
:bug: Remove duplicate component context menu item

### PR DESCRIPTION
**Pull request by pabloalba**

### CODE REVIEW

File: frontend/src/app/main/data/workspace/selection.cljs [hunk 1]
```diff
@@ -565,7 +565,9 @@
 (defn calc-duplicate-delta
   [obj state objects]
   (let [{:keys [id-original id-duplicated]}
-        (get-in state [:workspace-local :duplicated])]
+        (get-in state [:workspace-local :duplicated])
+        move? (and (cph/frame-shape? obj)
+                   (not (:component-root? obj)))]
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace/selection.cljs]: Perhaps what you want here is `(not (ctk/component-head obj))`

File: frontend/src/app/main/data/workspace/selection.cljs [hunk 2]
```diff
@@ -593,6 +595,7 @@
   (ptk/reify ::duplicate-selected
     ptk/WatchEvent
     (watch [it state _]
+      (prn "duplicate-selected" move-delta?)
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace/selection.cljs]: prn

File: frontend/src/app/main/data/workspace/selection.cljs (hunk 1 above)
Comment (pabloalba) [reply frontend/src/app/main/data/workspace/selection.cljs]: Almost! It's `(not (ctk/instance-head? obj))`

File: frontend/src/app/main/data/workspace/selection.cljs (hunk 2 above)
Comment (pabloalba) [reply frontend/src/app/main/data/workspace/selection.cljs]: Fixed

**Pull request merged**